packages/@azure/servicebus/data-plane/lib/clientEntityContext.ts

```diff
@@ -113,14 +118,23 @@ export namespace ClientEntityContext {
       entityPath: entityPath,
       requestResponseLockedMessages: new ConcurrentExpiringMap<string>(),
       isSessionEnabled: !!options.isSessionEnabled,
-      messageSessions: {}
+      messageSessions: {},
+      expiredMessageSessions: {}
     };
 
     (entityContext as ClientEntityContext).sessionManager = new SessionManager(
       entityContext as ClientEntityContext
     );
 
     (entityContext as ClientEntityContext).getReceiver = (name: string, sessionId?: string) => {
+      if (sessionId != undefined && entityContext.expiredMessageSessions[sessionId] === true) {
+        // TODO: Use an error constructor utility which could also serve as interface documentation for all types of errors thrown
```

Comment:
Let's file issues for these TODO's instead of putting them in the code.

Reply:
@ramya0820  Logged #1145, you can remove the TODO comment.

Reply:
updated